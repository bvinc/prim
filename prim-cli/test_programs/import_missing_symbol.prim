import geom.point.DoesNotExist

fn main() {
    println(0)
}
