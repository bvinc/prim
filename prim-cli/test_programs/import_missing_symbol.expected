ERROR: Module 'geom.point' does not define 'DoesNotExist'
